ansible/README.test.md

```diff
@@ -222,6 +222,11 @@ ansible-playbook test_sonic.yml -i {INVENTORY} --limit {DUT_NAME} -e testcase_na
 ```
 - Requires switch connected to a PTF testbed
 
+##### Portstat test
+```
+ansible-playbook test_sonic.yml -i {INVENTORY} -e testcase_name=portstat -e testbed_name={TESTBED_NAME}
```

Comment:
A minor update.
kindly add the **"--limit"** as an option providing flexibility to users to select the DUT  they want to use to run tests.

```ansible-playbook test_sonic.yml -i {INVENTORY} --limit {DUT_NAME} -e testcase_name=portstat -e testbed_name={TESTBED_NAME}```